### DISCUSSION TITLE
Is any open protocol supported which is based on nrf24l01 chip.

### DISCUSSION THREAD

**Discussion by sanjuchopracool** [pascallanger/DIY-Multiprotocol-TX-Module#609] [Q&A] [answered]
I am working on a receiver module based on nrf24, Is there any open protocol based on nrf24 which support  8+ channels and telemetry ?

**Comment by pascallanger** [comment, marked as answer]
Yes the Cabell protocol with an open source receiver too. Details here: https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Protocols_Details.md#cabell---34
You also have the LOLI protocol (8 channels but with features and open source RX too): https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Protocols_Details.md#LOLI---82
